benchmarks/dynamo/distributed.py

```diff
@@ -38,6 +49,7 @@ def run_model(args, model, inputs, key):
     setup(rank, world_size)
     if args.device == "cuda":
         # needed for FSDP
+        print(f"rank is {rank}, {type(rank)}")
```

Comment:
nit: maybe remove the print?

Reply:
thanks! i was being annoyed by that print and kept forgetting to fix it.